### ISSUE TITLE
RFE : dynamic # of columns

### ISSUE THREAD

**Issue by MagicFab** [millette/gatsby-starter-location-github#34]
When using narrow displays the columns are adjusted from 3 to 2 then 1 etc.

Similarly it would be useful to have the columns # increse when having a wide screen (for example HD / 4K).

**Comment by millette** [comment]
If someone want to donate a 4K monitor, I will be glad to work on this :-) But more seriously, I might expand the view to the whole screen, making room for maybe 5 columns in all (HD). For more (4K), it might come with version 2.

**Comment by MagicFab** [comment]
You can simulate 4K or any other resolution by scaling, check [this example](https://www.reddit.com/r/thinkpad/comments/2g933a/howto_scale_any_1366x768_display_to_1080p/).
Careful what you wish for, I hope your PC can drive 4K :)

**Comment by millette** [comment]
Yeah, I really don't want a 4K monitor, don't want to give anyone ideas. Thanks for the pointer.

**Comment by millette** [comment]
For wider screens, see if putting the search options in a sticky column would help.

Instead of

```
SEARCH OPTIONS
```
----
```
SEARCH RESULTS
```

It would be

```
SEARCH OPTIONS     |   SEARCH RESULTS
```
